source/unified-test-format/unified-test-format.rst

```diff
@@ -2775,15 +2849,16 @@ client. If the list of expected events is empty, the test runner MUST assert
 that no events were observed on the client. The process for matching events is
 described in `expectedEvent`_.
 
-If `test.outcome <test_outcome_>`_ is specified, for each `collectionData`_
-therein the test runner MUST assert that the collection contains exactly the
-expected data. The test runner MUST query each collection using the internal
-MongoClient, an ascending sort order on the ``_id`` field (i.e. ``{ _id: 1 }``),
-a "primary" read preference, a "local" read concern. When comparing collection
-data, the rules in `Evaluating Matches`_ do not apply and the documents MUST
-match exactly; however, test runners MUST permit variations in document key
-order or otherwise normalize the documents before comparison. If the list of
-documents is empty, the test runner MUST assert that the collection is empty.
+If `test.outcome <test_outcome_>`_ is specified, for each
+`outcomeCollectionData`_ therein the test runner MUST assert that the
+collection contains exactly the expected data. The test runner MUST query
+each collection using the internal MongoClient, an ascending sort order on
+the ``_id`` field (i.e. ``{ _id: 1 }``), a "primary" read preference, a
```

Comment:
This predates your PR, but since you're changing this please insert "and" for `, and a "local" read concern.`.

Reply:
Done.